manager/volume.go

```diff
@@ -468,6 +468,15 @@ func (m *VolumeManager) Expand(volumeName string, size int64) (v *longhorn.Volum
 
 	size = util.RoundUpSize(size)
 
+	if v.Spec.Size >= size {
+		logrus.Infof("Volume %v expansion is not necessary since current size %v >= %v", v.Name, v.Spec.Size, size)
```

Comment:
How's the modification?
```suggestion
		logrus.Infof("Volume %v expansion is not allowable since current size %v >= %v", v.Name, v.Spec.Size, size)
```

Reply:
It is good like denying a shrink operation.
Modified. Thanks